storage-proofs/porep/src/nse/vanilla/proof_scheme.rs

```diff
@@ -339,7 +349,9 @@ impl<'a, 'c, Tree: 'static + MerkleTreeTrait, G: 'static + Hasher> ProofScheme<'
         _requirements: &ChallengeRequirements,
         _partitions: usize,
     ) -> bool {
-        todo!()
+        // TODO:
```

Comment:
This is still missing and needs to be addressed, although not necessarily in this circuit PR.

Reply:
Yes, lets do this at a later point, I only switched to a manual todo, to be able to test the rest.